PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1400,10 +1400,13 @@ void Notepad_plus::removeEmptyLine(bool isBlankContained)
 		env._str2Search = TEXT("^$(\\r\\n|\\r|\\n)");
 	}
 	env._str4Replace = TEXT("");
-    env._searchType = FindRegex;
-
-	_findReplaceDlg.processAll(ProcessReplaceAll, &env, true);
-
+	env._searchType = FindRegex;
+	auto mainSelStart = _pEditView->execute(SCI_GETSELECTIONSTART);

```

Comment:
Is it nicer to do this here instead?
`Sci_CharacterRange mainSelection = _pEditView->getSelection();`
And then work with that in the next lines?
Is it dangerous to comment on one's own code?  Is that asking for trouble? :-)

Reply:
> Is it nicer to do this here instead?:
Sci_CharacterRange mainSelection = _pEditView->getSelection();

Yes, why not?

> Is it dangerous to comment on one's own code? Is that asking for trouble? :-)

Is it a philosophical question? Am I supposed to answer this question?  :)